include/mbedtls/arc4.h

```diff
@@ -32,6 +32,8 @@
 
 #include <stddef.h>
 
+#define MBEDTLS_ERR_ARC4_HW_ACCEL_FAILED                  -0x0019  /**< ARC4 hardware accelerator failed */
```

Comment:
This error code is not mentioned in `error.h`, which has no entry for ARC4 at all.